concrete-core/docs/user/introduction.md

```diff
@@ -0,0 +1,72 @@
+
+## Quick start
+
+Here is a quick example of how the library can be used to encrypt an integer and decrypt it:
+
+```rust
+extern crate concrete_core;
+extern crate concrete_commons;
+
+use concrete_commons::dispersion::Variance;
+use concrete_commons::parameters::LweDimension;
+use concrete_core::prelude::*;
+
+// DISCLAIMER: the parameters used here are only for demo purposes, and are not secure.
+let lwe_dimension = LweDimension(630);
+// Here a hard-set encoding is applied (shift by 20 bits)
+let input = 3_u32 << 20;
+let noise = Variance(2_f64.powf(-25.));
+
+let mut engine = CoreEngine::new().unwrap();
+let key: LweSecretKey32 = engine.create_lwe_secret_key(lwe_dimension).unwrap();
+let plaintext = engine.create_plaintext(&input).unwrap();
+let ciphertext = engine.encrypt_lwe_ciphertext(&key, &plaintext, noise).unwrap();
+
+let decrypted_plaintext = engine.decrypt_lwe_ciphertext(&key, &ciphertext).unwrap();
+
+engine.destroy(key).unwrap();
+engine.destroy(plaintext).unwrap();
+engine.destroy(ciphertext).unwrap();
+engine.destroy(decrypted_plaintext).unwrap();
+```
+
+## Audience
+
+The goal of `concrete-core` is to be the intermediate layer where library writers and compiler makers, can
+access a breadth of fast primitives developed by researchers and hardware designers. As such, this guide
+contains two kinds of tutorials geared towards this two categories of users.
+
+### Contributors to the public API
+
+If you are a researcher working on a fast FHE algorithm, or a hardware designer who want to accelerate an
+existing algorithm, you can contribute your code to `concrete-core`. What this gives you is:
++ The `concrete-benchmark` application, which allows to benchmark your implementation in minutes, and compare
+the results with the existing implementations.
++ The `concrete-test` application, which allows to easily test your implementation, to verify that your
+implementation is correct.
++ Exposure to external users, which can easily access your implementation, and integrate it within their
+library or compiler.
+
+If you are interested in contributing new algorithms, or new hardware accelerations to `concrete-core`,
+proceed to the [Contributor Guide](architecture.md) section !
+
+### Consumers of the public API
+
+If you are a writing a library or a compiler for FHE, you can rely on the public API of `concrete-core` to
+access a large set of homomorphic operators. To properly use those operators though, you have to know your way
+around FHE. The API gives you the freedom to control multiple parameters, which can lead to less than 128
+bits of security if chosen incorrectly. As a rule of thumb, if you did not carefully study the mathematical
+foundations of FHE, `concrete-core` is probably too low-level for you to use. Fortunately, we propose multiple
+libraries that build on top of `concrete-core` and which propose a safer API. To see which one best suits your
+needs, see the [concrete homepage](https://zama.ai/concrete).
```

Comment:
"To see which one best suits your needs" -> "To see which one suits your needs best"? Not sure